corehq/apps/userreports/tasks.py

```diff
@@ -46,7 +46,9 @@ def _get_db(doc_type):
 
 
 # This is intentionally not using magic to introspect the class from the name, though it could
+from corehq.apps.locations.models import Location
 _DOC_TYPE_MAPPING = {
     'XFormInstance': XFormInstance,
     'CommCareCase': CommCareCase,
+    'Location': Location
```

Comment:
@czue With the planned moving of forms and cases to SQL, have you given much thought to the changes necessary to UCR?  UCR currently uses couch documents as the primary row source, but most of the work happens on a single document, so I imagine it'd mostly be a matter of making the document processing work with a SQL model too (which sounds like it could get ugly).  Anyways, seems like this locations use-case would be a good pilot for that.